animeshon/tracker/v1alpha1/tracker.proto

```diff
@@ -22,72 +22,88 @@ service TrackerService {
 
   rpc GetTracker(GetTrackerRequest) returns (Tracker) {
     option (google.api.http) = {
-      get: "/v1alpha1/{name=users/*/trackers/*/*}"
+      get: "/v1alpha1/{name=users/*/trackers/*}"
+      additional_bindings {
+        get: "/v1alpha1/{name=audiences/*/trackers/*}"
+      }
     };
   }
```

Comment:
This will be solved inside the GraphQL custom resolver which will make it easier to query the tracker for a specific resource.
Whoever is using the REST API / gRPC API directly there will be additional documentation for this behavior.